src/model/Induction.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import v1 from 'uuid';
+import UniProtEvidenceTag from '../components/UniProtEvidenceTag';
+import { EvidenceType } from './types/modelTypes';
+import { FreeTextType } from './FreeText';
+
+type Comments = {
+  commentType: FreeTextType;
+  texts: [
+    {
+      value: string;
+      evidences?: EvidenceType[];
+    }
+  ];
+};
+
+type InductionData = {
+  comments?: Comments[];
+};
+
+type InductionProps = {
+  data: InductionData;
+};
+
+export const Induction: React.FC<InductionProps> = ({ data }) => {
+  const isValid = (d: Comments) =>
+    d.commentType === FreeTextType.INDUCTION && d.texts && d.texts.length > 0;
+  if (!data.comments || !data.comments.some(d => isValid(d))) {
+    return null;
+  }
+  const inductionData = data.comments.filter(d => isValid(d));
+  if (!inductionData || inductionData.length <= 0) {
```

Comment:
I think it's better to check the length with a positive condition. Either with `inductionData.length === 0` or `!inductionData.length > 0`, because the return value of the `length` function is an `unsigned int`, so it can't be negative.

Reply:
yep, totally agree. changed.